app/views/home/first_login_help.html.erb

```diff
@@ -1,49 +1,233 @@
+  different pseuds to post your works under the appropriate name while still 
+  managing them through the same account. You can %{manage_pseuds} through 
+  your profile page. For more information, please visit the 
+  %{pseuds_faq}.',
+  manage_pseuds: (link_to ts('manage your pseuds'), 
+                          user_pseuds_path(current_user)),
+  pseuds_faq: (link_to ts('Pseuds FAQ'), archive_faqs_path + '/pseuds'),
+  ).html_safe %></p>
+
+  <h3 class="heading" id="posting">
+    <%= ts('Posting Works') %>
+  </h3>
+  <p><%= ts('To open the Post New Work page, just select the %{post} link 
+  from the menu of the Post tab located at the top right of the page in the 
+  default browser skin. Visit the %{posting_faq} or check out our 
+  %{post_tutorial} for more information.',
+  post: (link_to ts('Post New'), new_work_path),
+  posting_faq: (link_to ts('Posting and Editing FAQ'), 
+                        archive_faqs_path + '/posting-and-editing'),
+  post_tutorial: (link_to ts('Tutorial: Posting a Work on AO3'), 
+                          archive_faqs_path + '/tutorial-posting-a-work-on-ao3')
+  ).html_safe %></p>
+
+  <h3 class="heading" id="browsing">
+    <%= ts('Browsing') %>
+  </h3>
+  <p><%= ts('You can start browsing works by going to the "%{fandoms}" tab at 
+  the top of any Archive page in the default skin and selecting either 
+  "%{all_fandoms}" or one of the subsets such as "%{movies}". Alternatively, you 
+  can use the %{search} to look for specific fandoms, works, or users. You can 
+  use the "Sort and Filter" form to narrow down the results in any fandom page. 
+  For more instructions, please visit %{search_faq} and the 
+  %{search_tutorial}.',
+  fandoms: (link_to ts('Fandoms'), menu_fandoms_path),
+  all_fandoms: (link_to ts('All Fandoms'), media_path),
+  movies: Media.find_by_name('Movies') ? (link_to ts('Movies'), 
+                                         medium_fandoms_path(
+                                           Media.find_by_name('Movies'))) :
+                                         ts('Movies'),
+  search: (link_to ts('search feature'), search_works_path),
+  search_faq: (link_to ts('Search and Browse FAQ'), 
+                       archive_faqs_path + '/search-and-browse'),
+  search_tutorial: (link_to ts('Searching and browsing tutorial'), 
+                            admin_posts_path + '/259'),
+  ).html_safe %></p>
+
+  <h3 class="heading" id="tags">
+    <%= ts('Tags') %>
+  </h3>
+  <p><%= ts('All tags on the Archive—including those for fandoms, relationships, 
+  and characters—start out user-created. You can always use the existing tags by 
+  selecting from the autocomplete list. If you cannot find the tags you want to 
+  use, feel free to create new tags for your works. Behind the scenes, our tag 
+  wrangling team will match the tags up with their synonyms, so that people can 
+  find your work whether you tag it "<abbr>AMTDI</abbr>", "Aliens Made Them Do 
```

Comment:
I wouldn't use a title attribute here -- partly to make it match the tag, partly to make sure screenreaders don't confuse someone by reading "Aliens Made Them Do It, Aliens Made Them Do It, or Sex Pollen." But consistent use of the <abbr> tag is a good idea even without the title attribute, since it can be used for styling or JavaScript or whatever else down the road. (It's also Doc's policy to use the tag in most cases.)